ui/src/app/base/components/PowerTypeFields/IPMIPowerFields/IPMIPowerFields.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import { Formik } from "formik";
+
+import IPMIPowerFields, {
+  NONE_WORKAROUND_VALUE,
+  WORKAROUNDS_FIELD_NAME,
+} from "./IPMIPowerFields";
+
+import type { PowerField } from "app/store/general/types";
+import { PowerFieldType } from "app/store/general/types";
+import { powerField as powerFieldFactory } from "testing/factories";
+
+let workaroundsField: PowerField;
+beforeEach(() => {
+  workaroundsField = powerFieldFactory({
+    field_type: PowerFieldType.MULTIPLE_CHOICE,
+    label: "Workaround flags",
+    name: WORKAROUNDS_FIELD_NAME,
+    choices: [
+      ["one", "One"],
+      [NONE_WORKAROUND_VALUE, "None"],
+    ],
+  });
+});
+
+it("does not render the 'None' choice for the workaround flags field", async () => {
```

Comment:
I think it's probably also worth adding tests for when it's not a multi-choice and also for selecting the "None" choice when all other fields are unselected.